tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js

```diff
@@ -59,9 +60,20 @@ module.exports = class customerSettings extends BOBasePage {
    * @param toEnable
    * @returns {Promise<string>}
    */
-  async setPartnerOfferStatus(toEnable = true){
+  async setPartnerOfferStatus(toEnable = true) {
     await this.waitForSelectorAndClick(this.enablePartnerOfferLabel.replace('%TOGGLE', toEnable ? 1 : 0));
     await this.clickAndWaitForNavigation(this.saveGeneralFormButton);
     return this.getTextContent(this.alertSuccessBlock);
   }
+
+  /**
+   * Set B2B mode status
+   * @param toEnable
+   * @returns {Promise<string>}
+   */
+  async setB2BModeStatus(toEnable = true) {
```

Comment:
This is the same code as setPartnerOfferStatus, only the selector is different. Maybe create a generic function with 
`this.enableB2BModeToggle` as parameter

Reply:
we don't want to use or call a selector in tests files but I can't create a generic function.

Reply:
Fixed, WDYT @PierreRambaud @NeOMakinG ?